guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>The parameters required are: </p><ul><li><code>data</code>&nbsp;- the data object prepared using&nbsp;<code>prepare_data</code></li><li><code>model</code>&nbsp;- the name of the models from the list of supported models.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>